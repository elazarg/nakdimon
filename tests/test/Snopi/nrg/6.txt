הַדֶּרֶךְ האולטימטיבית לְהַכִּיר אֶת סצינת הָאוּמָּנוּת הַמְּקוֹמִית הִיא לְצֵאת לְסִיּוּר אוּמָּנוּת שֶׁמַּעֲבִירָה יָעֵל לרנר בְּכָל יוֹם שְׁלִישִׁי.
מִכֵּיוָון שֶׁזֹּאת הַתְּקוּפָה הַחַמָּה בְּעִיר, יוֹצְאִים לַסִּיּוּר במיניבוס מְמוּזָּג.
מַתְחִילִים בְּמוּזֵיאוֹן וְגַלֶּרְיַית אַיֶּלֶת, לִטְעִימָה קַלָּה מֵאוּמָּנוּת ממוסדת, וְאָז יוֹצְאִים לְמַסָּע בַּעֲקֵבוֹת אָומָּנִים מְקוֹמִיִּים וּמַבְטִיחִים.
בֵּין הַיֶּתֶר הַסִּיּוּר עוֹצֵר בַּבַּיִת פיליפ מוֹרַיי - מהוותיקים במבני אַיֶּלֶת, שֶׁפּוֹעַל מ-1957.
הַמִּבְנֶה שִׁימֵּשׁ לָאוֹרֶךְ הַשָּׁנִים כְּהֵיכַל מוּזִיקָה וְתֵיאַטְרוֹן.
לָאַחֲרוֹנָה הַמָּקוֹם שׁוּפַּץ, וְכָעֵת מִתְקַיֶּימֶת בּוֹ תַּעֲרוּכָה שֶׁל נבו אֹפֶק, בָּחוּר צָעִיר בַּעַל לקות רְאִיָּיה, שֶׁנִּמְצָא עַל הָרֶצֶף האוטיסטי.
נבו הוּא דּוּגְמָה לָאָומָּן שֶׁמַּצְלִיחַ לַמְרוֹת כָּל הַקְּשָׁיִים.
אֶת הָאוּמָּנוּת שֶׁלּוֹ הוּא יוֹצֵר בַּדַּיְיקָנוּת וּבִרְגִישׁוֹת, וְהַמִּתְבּוֹנֵן בָּהּ לֹא יָכוֹל לָדַעַת שֶׁמֵּאֲחוֹרֶיהָ עוֹמֵד אָדַם עִם סִיפּוּר חַיִּים לֹא פָּשׁוּט בַּכְּלָל.