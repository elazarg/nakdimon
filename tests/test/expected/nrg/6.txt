הַדֶּרֶךְ הָאוּלְטִימָטִיבִית לְהַכִּיר אֶת סְצֵינַת הָאוֹמָּנוּת הַמְּקוֹמִית הִיא לָצֵאת לְסִיּוּר אוֹמָּנוּת שֶׁמַּעֲבִירָה יָעֵל לֶרְנֶר בְּכָל יוֹם שְׁלִישִׁי.
מִכֵּיוָון שֶׁזֹּאת הַתְּקוּפָה הַחַמָּה בָּעִיר, יוֹצְאִים לְסִיּוּר בְּמִינִיבּוּס מְמוּזָּג.
מַתְחִילִים בְּמוּזֵיאוֹן וְגָלֶרְיַית אֵילַת, לִטְעִימָה קַלָּה מֵאוֹמָּנוּת מְמוּסֶּדֶת, וְאָז יוֹצְאִים לְמַסָּע בְּעִקְבוֹת אוֹמָּנִים מְקוֹמִיִּים וּמַבְטִיחִים.
בֵּין הַיֶּתֶר הַסִּיּוּר עוֹצֵר בְּבֵית פִילִיפּ מוֹרֵיי - מֵהַוָּותִיקִים בְּמִבְנֵי אֵילַת, שֶׁפּוֹעֵל מְ-1957.
הַמִּבְנֶה שִׁימֵּשׁ לְאוֹרֶךְ הַשָּׁנִים כְּהֵיכַל מוּזִיקָה וְתֵיאַטְרוֹן.
לָאַחֲרוֹנָה הַמָּקוֹם שׁוּפַּץ, וְכָעֵת מִתְקַיֶּימֶת בּוֹ תַּעֲרוּכָה שֶׁל נְבוֹ אֲפֵק, בָּחוּר צָעִיר בַּעַל לְקוּת רְאִיָּיה, שֶׁנִּמְצָא עַל הָרֶצֶף הָאוֹטִיסְטִי.
נְבוֹ הוּא דּוּגְמָה לְאוֹמָן שֶׁמַּצְלִיחַ לַמְרוֹת כָּל הַקְּשָׁיִים.
אֶת הָאוֹמָּנוּת שֶׁלּוֹ הוּא יוֹצֵר בְּדַיְיקָנוּת וּבִרְגִישׁוּת, וְהַמִּתְבּוֹנֵן בָּהּ לֹא יָכוֹל לָדַעַת שֶׁמֵּאֲחוֹרֶיהָ עוֹמֵד אָדָם עִם סִיפּוּר חַיִּים לֹא פָּשׁוּט בִּכְלָל.